cfgov/jinja2/v1/events/_event-detail.html

```diff
@@ -7,11 +7,11 @@ <h1 class="post_heading">
         </h1>
         <p class="post_dek">
         {% if (event_state == 'past') %}
-            {{ page.archive_body | striptags }}
```

Comment:
i don't even think strip tags is necessary here, could be fine with just `| safe`